controllers/comments.controller.js

```diff
@@ -1,10 +1,27 @@
-const { fetchArticleComments } = require('../models/comments.model');
+const {
+    fetchArticleComments,
+    addComment,
+    articleCheck
+} = require('../models/comments.model');
 
 exports.getArticleComments = (req, res, next) => {
     const { article_id } = req.params;
     fetchArticleComments(article_id)
         .then(comments => {
-            res.status(200).send({comments});
+            res.status(200).send({ comments });
         })
         .catch(next);
-};
\ No newline at end of file
+};
+
+exports.postComment = (req, res, next) => {
+    const { article_id } = req.params;
+    const commentData = req.body;
+    return articleCheck(article_id)
```

Comment:
this isn't really necessary as article_id is a foreign key on the comments table and therefore will have to exist.